src/components/ButtonPanel.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Button from './Button';
+
+const group1 = ['AC', '+/-', '%', '+'];
+const group2 = ['7', '8', '9', 'X'];
+const group3 = ['4', '5', '6', '-'];
+const group4 = ['1', '2', '3', '+'];
+const group5 = ['0', '.', '='];
+
+const renderGroup = group => <div>{group.map((name) => <Button name={name} />)}</div>;
```

Comment:
Lists in react need to have a unique `key` property in order to identify them for performance reasons. See [docs](https://reactjs.org/docs/lists-and-keys.html#keys). 
So let's add a `key` prop to the mapped component.